File: src/main/java/com/indiepost/dto/analytics/Trend.java
package com.indiepost.dto.analytics;

import com.indiepost.enums.Types.TimeDomainDuration;

import java.util.List;

/**
 * Created by jake on 17. 5. 24.
 */
public class Trend {

    private TimeDomainDuration duration;

    private List<TimeDomainStat> result;

    public TimeDomainDuration getDuration() {
        return duration;
    }

    public void setDuration(TimeDomainDuration duration) {
        this.duration = duration;
    }

    public List<TimeDomainStat> getResult() {
        return result;
    }

    public void setResult(List<TimeDomainStat> result) {
        this.result = result;
    }
}

File: src/main/kotlin/com/indiepost/dto/analytics/Trend.kt
package com.indiepost.dto.analytics

import com.indiepost.enums.Types.TimeDomainDuration

/**
 * Created by jake on 17. 5. 24.
 */
data class Trend(
        var duration: TimeDomainDuration? = null,

        var result: List<TimeDomainStat>? = null
)
